pkg/signature/kms/kms_test.go

```diff
@@ -1,35 +0,0 @@
-//
-// Copyright 2021 The Sigstore Authors.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package kms_test
-
-import (
-	"testing"
-
-	"github.com/sigstore/sigstore/pkg/signature/kms"
-	"github.com/sigstore/sigstore/pkg/signature/kms/aws"
-	"github.com/sigstore/sigstore/pkg/signature/kms/azure"
-	"github.com/sigstore/sigstore/pkg/signature/kms/hashivault"
-
-	"github.com/stretchr/testify/require"
-)
-
-func Test_SupportedProviders(t *testing.T) {
```

Comment:
Can we still have this test?

Reply:
This test would reintroduce all the newly split modules into dependencies of sigstore/sigstore. They would get trimmed out of built binaries since it's only a test dependency, but downstream `go.mod`s would still inherit these dependencies, and any source-level CVE warnings from GitHub for all transitive dependencies.

Reply:
Ah, that makes sense.